Dockerfiles/agent/test_image_contents.py

```diff
@@ -34,7 +34,7 @@ def test_files_absent(self):
             self.assertFalse(os.path.isfile(file), file + " should NOT be present")
 
     def test_files_checksums(self):
-        for file, digest in EXPECTED_CHECKSUMS.iteritems():
+        for file, digest in EXPECTED_CHECKSUMS.items():
```

Comment:
Does that work the same both in py2 and py3? I thought we used a lib to smooth it out.

Reply:
In python 2 it returns a list of key/value pairs and in python 3 it returns a view however these should work the same when iterating over. I guess `six.iteritems()` would look cleaner.